inc/features/blocks/experience/register.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Experience Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
+ */
+
+namespace Altis\Experiments\Features\Blocks\Experience;
+
+use const Altis\Experiments\ROOT_DIR;
+use function Altis\Experiments\Features\Blocks\get_block_settings;
+
+const BLOCK = 'experience';
+
+function setup() {
+	$block_data = get_block_settings( BLOCK );
+
+	// Queue up JS files.
+	add_action( 'enqueue_block_editor_assets', __NAMESPACE__ . '\\enqueue_assets' );
+
+	// Register the block.
+	register_block_type( $block_data['name'], [
+		'attributes' => $block_data['settings']['attributes'],
+		'render_callback' => __NAMESPACE__ . '\\render_block',
+	] );
+}
+
+function enqueue_assets() {
+	wp_enqueue_script(
+		'altis-experiments-features-blocks-experience',
+		plugins_url( 'build/features/blocks/experience.js', ROOT_DIR . '/plugin.php' ),
+		[
+			'wp-plugins',
+			'wp-blocks',
+			'wp-i18n',
+			'wp-editor',
+			'wp-components',
+			'wp-core-data',
+			'wp-edit-post',
+			'altis-experiments-features-blocks-variant',
+			// Exports the audience UI modal component.
+			'altis-analytics-audience-ui',
+		]
```

Comment:
Are we missing a version here? As below too

Reply:
Yeah, I might add the webpack manifest generator actually so the file name contains a hash of the current content